<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsWPilotHorisintalVerticalSteps</class>
 <widget class="QWidget" name="tlinsWPilotHorisintalVerticalSteps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupAxis">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Axis</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioHorisontal">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string> Horisontal/RA</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioVertical">
        <property name="font">
         <font>
          <italic>false</italic>
          <underline>false</underline>
         </font>
        </property>
        <property name="text">
         <string> Vertical/DC</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Direction</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="radioDirLeft">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Left/Down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDirRight">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Right/Up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="title">
      <string>Mode</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioModeSteps">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Steps</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioModeDegris">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Degris</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioModeAstro">
        <property name="font">
         <font>
          <italic>false</italic>
         </font>
        </property>
        <property name="text">
         <string>Astro</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
